target-platform/target-platform.target

```diff
@@ -45,6 +45,7 @@
 			<repository location="https://download.eclipse.org/cbi/updates/license/"/>
 			<unit id="org.eclipse.license.feature.group" version="0.0.0"/>
 		</location>
+		<location path="${eclipse_home}" type="Profile"/>
```

Comment:
Is this needed as part of this change?

Reply:
No this should not be part of the change...